aicsimageio/tests/test_aics_image.py

```diff
@@ -331,6 +331,35 @@ def test_channel_names(resources_dir, filename, expected_channel_names):
     assert str(f) not in [f.path for f in proc.open_files()]
 
 
+@pytest.mark.parametrize(
+    "filename, expected_sizes",
+    [
+        (PNG_FILE, (1.0, 1.0, 1.0)),
+        (TIF_FILE, (1.0, 1.0, 1.0)),
+        (CZI_FILE, (1.0833333333333333e-06, 1.0833333333333333e-06, 1.0)),
+        (OME_FILE, (1.0833333333333333, 1.0833333333333333, 1.0)),
+    ],
+)
+def test_physical_pixel_size(resources_dir, filename, expected_sizes):
+    # Get filepath
+    f = resources_dir / filename
+
+    # Check that there are no open file pointers after init
+    proc = Process()
+    assert str(f) not in [f.path for f in proc.open_files()]
```

Comment:
I don't think that this should necessarily be included in every test, because if something changes where this behavior is different it will make all of the tests fail erroneously

Reply:
Previously, these checks have caught when even calling `metadata` on the base reader kept a file pointer open. I guess the tests for metadata in general would catch this but I prefer to keep it in the case we change metadata retrieval?

Reply:
Ah, I see. Maybe it'd be good to have these wrapped in functions then if they're going to be used frequently throughout the tests?

Reply:
Yea there is definitely some test code de-duplication that could be done. I don't want to do it right now :upside_down_face: / at least in this PR.